db/migrate/20220628153154_add_team_leader_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTeamLeaderToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :team_leader, :boolean, default: false
```

Comment:
Not sure we needed this yet, but it's early days and I know you are working on the authorisation, so happy it is here. 👍